### ISSUE TITLE
Vulture Squadron error in TTT3 (low priority)

### ISSUE THREAD

**Issue by TheBlackxRanger** [emperorshammer/issues-incoming-queue#88]
Hey Team,

I was playing around in TTT3 just to see what would show if i pulled up a pilot thats on the M/FRG Phoenix / Vulture Squadron and received the following error. 

![image](https://user-images.githubusercontent.com/74426625/163035662-fdca8812-634d-42bb-a07d-e69b4944b9d0.png)

**Comment by SmTaUcM** [comment]
This might easily be fixed by changing Vulture squadron member's TTT position to RSV.
Otherwise we may need to add a Wing for Vulture to be hosted in and make the Phoenix a selectable ship in TTT via API.

This also raises the question of what a Vulture Squadron member's uniform should look like?

**Comment by TheBlackxRanger** [comment]
> This might easily be fixed by changing Vulture squadron member's TTT position to RSV. Otherwise we may need to add a Wing for Vulture to be hosted in and make the Phoenix a selectable ship in TTT via API.
> 
> This also raises the question of what a Vulture Squadron member's uniform should look like?

Reserve Dress Uni but with the Vulture squadron patch, if there is one? I think the dress uniform has the gray accents

Oops and while trying to load up a reservist TTT3 crashed on me 
![image](https://user-images.githubusercontent.com/74426625/163048035-f0e5e2c4-5257-4a65-9acb-1fc8586e6f4d.png)

![image](https://user-images.githubusercontent.com/74426625/163048073-fa7522f3-7925-4ed1-ad51-f3f6d90757dd.png)

**Comment by SmTaUcM** [comment]
Ah yes , I believe that we're aware of the reservist bug, that's also an API TTT position change that's required

**Comment by turtlexiv** [comment]
The uniform of a Reservist is better described as the 'standard' uniform.  Trim color changes based on assigned Wing.  Since the squadron is not part of a Wing, pilots assigned there will not wear colored trim.

There is no approved patch for Vulture.

@SmTaUcM we probably need to do a fix for TTT3 to allow the default uniform for a FM/FL/CMDR assigned to a Squadron that's not part of a Wing, since this a valid unit configuration.

**Comment by SmTaUcM** [comment]
Yep  I can code out the filters/restrictions where TTT3 requires ship and or Wing to ve selected. I'll have it just default to the standard trim when no Wing has been selected.

Transferring this issue to the main TTT repository
